app/views/admin/deploy_groups/show.html.erb

```diff
@@ -42,7 +42,7 @@
         <%= link_to "Deploy All Projects", deploy_all_admin_deploy_group_path(@deploy_group), class: "btn btn-default", data: { method: "post" } %>
         <%= link_to "Merge #{@deploy_group.stages.cloned.count} Cloned Stages",
                     merge_all_stages_admin_deploy_group_path(@deploy_group),
-                    class: "btn btn-default",
+                    class: "btn btn-default #{ !@deploy_group.stages.cloned.exists? ? 'disabled' : '' }",
```

Comment:
hehehe or remove `!` and flip the condition :)

possibly: `'disabled' unless xxx.exists?`